osidb/api_views.py

```diff
@@ -811,6 +819,49 @@ def destroy(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         return super().destroy(request, *args, **kwargs)
 
 
+class FlawAvailableView(
```

Comment:
nitpick: I think it makes more sense to use a function-based view in this case, you'd avoid a lot of boilerplate and in the end this is a single-method class